app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java

```diff
@@ -21,195 +22,212 @@
 import java.util.List;
 import timber.log.Timber;
 
-public class UploadMediaDetailAdapter extends RecyclerView.Adapter<UploadMediaDetailAdapter.ViewHolder> {
+public class UploadMediaDetailAdapter extends
+    RecyclerView.Adapter<UploadMediaDetailAdapter.ViewHolder> {
 
-    private List<UploadMediaDetail> uploadMediaDetails;
-    private Callback callback;
-    private EventListener eventListener;
+  private List<UploadMediaDetail> uploadMediaDetails;
+  private Callback callback;
+  private EventListener eventListener;
 
-    private HashMap<AdapterView, String> selectedLanguages;
-    private final String savedLanguageValue;
+  private HashMap<AdapterView, String> selectedLanguages;
+  private final String savedLanguageValue;
 
-    public UploadMediaDetailAdapter(String savedLanguageValue) {
-        uploadMediaDetails = new ArrayList<>();
-        selectedLanguages = new HashMap<>();
-        this.savedLanguageValue = savedLanguageValue;
-    }
+  public UploadMediaDetailAdapter(String savedLanguageValue) {
```

Comment:
These lines shouldn't be edited for this PR. Kindly please revert them.

Reply:
thanks for your review. I auto indent with android studio and that the result. Should i modify anyway ?

Reply:
Yes, please :/ Otherwise it wont be similar with other files.